include/mbedtls/ecp.h

```diff
@@ -280,15 +280,16 @@ mbedtls_ecp_group;
 
 #if !defined(MBEDTLS_ECP_FIXED_POINT_OPTIM)
 /*
- * Trade memory for speed on fixed-point multiplication.
+ * Trade ROM or memory for speed on fixed-point multiplication.
  *
  * This speeds up repeated multiplication of the generator (that is, the
  * multiplication in ECDSA signatures, and half of the multiplications in
  * ECDSA verification and ECDHE) by a factor roughly 3 to 4.
  *
- * The cost is increasing EC peak memory usage by a factor roughly 2.
+ * The cost is increasing ROM usage for well-known curve or increasing EC peak
+ * memory usage for user-defined curve by a factor roughly 2.
```

Comment:
Note: I don't think we support user-defined curves. (That's not part of the official API anyway.)

Reply:
Since users can directly modify the curve parameters of mbedtls_ecp_group to implement custom curves, I mistakenly believe that this is official support.

Okay, I will move it.

Reply:
You're reasoning is correct, and to be honest currently it's neither officially supported, nor officially not supported. But in Mbed TLS 3.0 we're going to make it clear that directly accessing the members of a structure is not something that's supported. And this PR is likely to land only in the 3.0 branch, as it's changing the observable behaviour of an existing option.

Reply:
The comment has been fixed.